modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -119,6 +119,7 @@ public PythonClientCodegen() {
         languageSpecificPrimitives.add("object");
         // TODO file and binary is mapped as `file`
         languageSpecificPrimitives.add("file");
+        languageSpecificPrimitives.add("bytes");
```

Comment:
No longer needed because we are only concerned with bytes in the python layer when grabbing the raw resonse?

Reply:
Adding the type here without handling it in python-experimental beans that we will have a new unhandled type.
Can you defer this to the next PR?